### DISCUSSION TITLE
How do I know I am runnig the new forked version?

### DISCUSSION THREAD

**Discussion by oparada1988** [streamdeck-linux-gui/streamdeck-linux-gui#24] [Q&A]
Hi Guys, 

thank you for forking this project and keeping it alive. I was using the original version and moved to this new one. I uninstalled it and ran the steps in the installation documentation for this new one, however my streamdeck still reads 2.0.15

![Screenshot from 2023-07-30 14-49-13](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/4031078/7a8846df-abb6-4fc7-a04a-2136de1d59ac)

How can I tell if I am running the new release? I used the command "python3 -m pip install streamdeck-linux-gui --user" as per the guide.
![Screenshot from 2023-07-30 14-50-52](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/4031078/b5523252-4b63-4f1f-8ece-c8ca51cd1241)

Thanks again!

**Comment by Danie10** [comment]
Maybe it had changed for you after a reboot? Mine is showing 'Version devel' but I'd certainly like to rather see an actual number version with devel added if needs be eg. 'v3.0.1 Devel'.

![Screenshot_20230802_175739](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/1153726/89058f1b-80bf-4880-9cef-d86ffe11a8ec)

**Reply by Data-007** [reply]
Here's mine I don't know where the actual number comes from.
I am supposed to be on the new new version.
![StreamdeckUIAbout](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/53057579/77ba774f-cd6f-49f0-a64e-849ae5ffc0f8)

**Comment by Danie10** [comment]
Only way I really know is I have upgraded to Pillow 10.0.0 and it now works fine.

**Comment by coolapso** [comment]
there was actually a bug related with that, fixed on devleop branch: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/issues/36 

But as long as you install the package `streamdeck-linux-gui` you will be for sure on the new one as they are entirely different packages even tho the code and the application is the same.

**Comment by Ghr00t** [comment]
I had this exact question.  I'd love to see the version number updated as confirmation i did things correctly.  I'm relatively new to linux, and getting my streamdeck working was one of my first 'problems'  I tackled.

**Reply by coolapso** [reply]
if you are using the latest version you should see  3.1.0

**Reply by Ghr00t** [reply]
I removed everything and reinstalled it.  Still showing :

streamdeck 0.9.4
pyside6 6.5.2
pillow 9.5.0
pynput 1.7.6

However after entering the installation command "python3 -m pip install streamdeck-linux-gui --user", it shows:

Installing collected packages: streamdeck-linux-gui
Successfully installed streamdeck-linux-gui-3.0.1

So i'm not sure where im supposed to see 3.1.0 anywhere.

**Reply by oparada1988** [reply]
Im having the same issue, I uninstalled everything and reinstalled it again. But 3.0.1 keeps getting installed. As a work around I downloaded the built files and overwrote the ones in "~/.local/lib/python3.10/site-packages" and i got the new release running now

**Reply by coolapso** [reply]
I suspect you still have the old package (streamdeck-ui) installed, you might have to uninstall it first.
